packages/plugin/webpack/test/WebpackConfig_spec.ts

```diff
@@ -27,51 +27,6 @@ const sampleWebpackConfig = {
 };
 
 describe('WebpackConfigGenerator', () => {
-  describe('rendererTarget', () => {
```

Comment:
Is there a replacement for these tests, we still decide targets based on entrypoint objects right?